_inc/client/security/index.jsx

```diff
@@ -31,54 +30,58 @@ export const Security = React.createClass( {
 			isUnavailableInDevMode: this.props.isUnavailableInDevMode
 		};
 
-		const found = {
-			protect: this.props.isModuleFound( 'protect' ),
-			sso: this.props.isModuleFound( 'sso' )
-		};
+		const foundProtect = this.props.isModuleFound( 'protect' ),
+			foundSso = this.props.isModuleFound( 'sso' ),
+			foundAkismet = this.props.isPluginActive( 'akismet/akismet.php' ),
+			foundBackups = this.props.isPluginActive( 'vaultpress/vaultpress.php' );
 
 		if ( ! this.props.searchTerm && ! this.props.active ) {
 			return null;
 		}
 
-		if ( ! found.sso && ! found.protect ) {
+		if (
+			! foundSso &&
+			! foundProtect &&
+			! foundAkismet &&
+			! foundBackups
+		) {
 			return null;
 		}
 
-		const backupSettings = (
-			<BackupsScan
-				{ ...commonProps }
-			/>
-		);
-
-		let akismetSettings = '';
-		if ( this.props.isPluginActive( 'akismet/akismet.php' ) ) {
-			akismetSettings = (
-				<div>
-					<Antispam
-						{ ...commonProps }
-					/>
-					<QueryAkismetKeyCheck />
-				</div>
-			);
-		}
-		const protectSettings = (
-			<Protect
-				{ ...commonProps }
-			/>
-		);
-		const ssoSettings = (
-			<SSO
-				{ ...commonProps }
-			/>
-		);
 		return (
 			<div>
 				<QuerySite />
-				<QuerySitePlugins />
-				{ ( found.protect || found.sso ) && backupSettings }
-				{ ( found.protect || found.sso ) && akismetSettings }
-				{ found.protect && protectSettings }
-				{ found.sso && ssoSettings }
+				{
```

Comment:
This works fine in cases of filtered out modules, but has one caveat when you search. You can see in the previous versions that Backups and Akismet were only displayed when either protect or sso were "found". That prevented those cards from showing up in every search, as they do now. 

If we revert to previous logic, that problem would be solved, but we will have another problem - Backup and Akismet will disappear when Protect and SSO are both filtered out. But having Protect and SSO filtered out is a much less frequently happening edge case.

In the future we'll make sure we show and hide things based on features, not modules, and this problem will be solved entirely.